### PR TITLE
15 product modal

### PR DESCRIPTION
**Pull request by dousdogukan**
### Description

implement CSelect
implement AddProductModal
implement EditProductModal
move product page to containers
add CTokenLabel to Product page

### Checklist

- [ ] Tested changes locally.
- [ ] Ensured code follows project standards.
- [ ] Updated documentation.
- [ ] Added necessary tests.

### Additional Notes
for #15

### CODE REVIEW

File: src/components/CProductItemCard/index.tsx [hunk 1]
```diff
@@ -2,9 +2,11 @@ import { StaticImageData } from 'next/image';
 
 import CItemCard from '../CItemCard';
 import CMethods, { MethodType } from '../CMethods';
+import CTokenLabel from '../CTokenLabel';
 
 export type TokensType = {
-  symbol: string;
```
Comment (ruveyda-kisla) [inline src/components/CProductItemCard/index.tsx:7]: this causes conflict with new PR

File: src/constants/productsList.tsx [hunk 2]
```diff
@@ -5,90 +5,45 @@ import hoodiImage from 'public/images/hoodie.png';
 
 const tokensUser: TokensType[] = [
   {
-    symbol: 'dai',
+    value: 'dai',
```
Comment (ruveyda-kisla) [inline src/constants/productsList.tsx:8]: causes Conflict

**Review by ruveyda-kisla — CHANGES_REQUESTED** [review]
bring symbols back

File: src/components/CButtonGroup/index.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  tabs: CButtonGroupType[];
+  defaultSelectedTabs?: string[];
+  selectedMethod?: string[];
+  onChange?: (value: string[]) => void;
+}
+
+const CButtonGroup = ({
+  tabs,
+  defaultSelectedTabs = [],
+  selectedMethod = [],
```
Comment (ceydae) [inline src/components/CButtonGroup/index.tsx:19]: rename to `value`

File: src/components/CButtonGroup/index.tsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+interface CButtonGroupProps {
+  tabs: CButtonGroupType[];
+  defaultSelectedTabs?: string[];
+  selectedMethod?: string[];
+  onChange?: (value: string[]) => void;
+}
+
+const CButtonGroup = ({
+  tabs,
+  defaultSelectedTabs = [],
```
Comment (ceydae) [inline src/components/CButtonGroup/index.tsx:18]: rename to `defaultValue`

**Review by ceydae — CHANGES_REQUESTED** [review]

File: src/components/CProductItemCard/index.tsx (hunk 1 above)
Comment (dousdogukan) [reply src/components/CProductItemCard/index.tsx:7]: I can't change it because of the select component

**Review by ruveyda-kisla — APPROVED** [review]

**Review by ceydae — APPROVED** [review]

**Review by ruveyda-kisla — APPROVED** [review]

**Pull request merged**